res_interpreter_language/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import res_interpreter_language  # noqa:F401
```

Comment:
Same here

Reply:
It tells the Linter to ignore the Fault F401 (Imported function is not used)